bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gpi/Gpi.java

```diff
@@ -792,6 +818,10 @@ public static Apodizer valueOf(String name, Apodizer value) {
             }
             return None.instance();
         }
+
+        public static Apodizer[] validValues() {
+            return Arrays.stream(values()).filter(a -> !a.isObsolete()).toArray(Apodizer[]::new);
+        }
```

Comment:
No reason really, I'll adjust to make them consistent. `nonObsoleteValues` seems the clearer